### ISSUE TITLE
Expect more training details

### ISSUE THREAD

**Issue by lingtianxia123** [bcmi/Object-Shadow-Generation-Dataset-DESOBA#12]
Thanks for sharing the dataset and code. The test results I obtained by trained SGRNet from scratch were different from the results of the paper. The GRMSE is about 5.5 and LRMSE is about 71 on BOS test set. Are there more training details.

**Comment by hy-zpg** [comment]
we have updated training codes: using more training data by setting the foreground object number as 1~3 (see src/data/shadowparam_dataset.py), and training the mode for about 800 epochs.
